src/WpfMath/TexFormulaParser.cs

```diff
@@ -247,6 +247,29 @@ private TexFormula Parse(SourceSpan value, ref int position, bool allowClosingDe
             return formula;
         }
 
+        private static TexFormula ParseChars(SourceSpan value, string textStyle)
+        {
+            var localPosition = 0;
+            return ParseChars(value, ref localPosition, textStyle);
+        }
+
+        private static TexFormula ParseChars(SourceSpan value, ref int position, string textStyle)
```

Comment:
Having that said, I think that this overload is redundant.

We'll never want to pass `ref int position` to `ConvertRawText`, because nobody should care about the extracted position from the text; it will always convert the text completely, so `position` will always point to the `value` end in the source text.

So, let's merge these overloads and get rid of `ref int position`?